src/Networks/Blockcore.Networks.HomeCoin/HomeCoinSetup.cs

```diff
@@ -0,0 +1,123 @@
+using System;
+using System.Collections.Generic;
+using Blockcore.Consensus.Checkpoints;
+using HomeCoin.Networks.Setup;
+using NBitcoin;
+
+namespace HomeCoin
+{
+    internal class HomeCoinSetup
+    {
+        internal static HomeCoinSetup Instance = new HomeCoinSetup();
+
+        internal CoinSetup Setup = new CoinSetup
+        {
+            FileNamePrefix = "homecoin",
+            ConfigFileName = "homecoin.conf",
+            Magic = "48-4F-4D-45",
+            CoinType = 100500, //0x80018894// SLIP-0044: https://github.com/satoshilabs/slips/blob/master/slip-0044.md,
+            PremineReward = 1000000,
+            PoWBlockReward = 10,
+            PoSBlockReward = 10,
+            LastPowBlock = 100500,
+            MaxSupply = 21000000,
+            GenesisText = "I'd like to know what this whole show is all about before it's out.",
+            TargetSpacing = TimeSpan.FromSeconds(64),
+            ProofOfStakeTimestampMask = 0x0000000F,
+            PoSVersion = 3
+        };
+
+        internal NetworkSetup Main = new NetworkSetup
+        {
+            Name = "HomeCoinMain",
+            RootFolderName = "homecoin",
+            CoinTicker = "HOME",
+            DefaultPort = 33331,
+            DefaultRPCPort = 33332,
+            DefaultAPIPort = 33333,
+            DefaultSignalRPort = 33334,
+            PubKeyAddress = 40, // B https://en.bitcoin.it/wiki/List_of_address_prefixes
```

Comment:
The comment should be updated to be "H" so anyone viewing code will get the correct prefix in their heads :)